reo/nested_inputs.py

```diff
@@ -1833,6 +1907,307 @@ def list_of_dict(input):
+        },                         
+        "isentropic_efficiency": {
+          "type": "float", "min": 0.0, "max": 1.0,
+          "description": "Steam turbine isentropic efficiency - uses inlet T/P and outlet T/P/X to get power out"
+        },
+        "gearbox_generator_efficiency": {
+          "type": "float", "min": 0.0, "max": 1.0,
+          "description": "Conversion of steam turbine shaft power to electric power based on combined gearbox and electric generator efficiency"
+        },
+        "net_to_gross_electric_ratio": {
+          "type": "float", "min": 0.0, "max": 1.0,
+          "description": "Conversion of gross electric power to net power which can account for e.g. pump power"
+        },                
+        "installed_cost_us_dollars_per_kw": {
+          "type": "float", "min": 0.0, "max": 100000.0,
+          "description": "Electric power-based cost"
+        },
+        "om_cost_us_dollars_per_kw": {
+          "type": "float", "min": 0.0, "max": 5000.0,
+          "description": "Electric power-based fixed O&M cost"
+        },
+        "om_cost_us_dollars_per_kwh": {
+          "type": "float", "min": 0.0, "max": 100.0,
+          "description": "Electric energy-based variable O&M cost"
+        },
+        "can_net_meter": {
+          "type": "bool",
+          "default": False,
+          "description": "True/False for if technology has option to participate in net metering agreement with utility. Note that a technology can only participate in either net metering or wholesale rates (not both)."
+        },
+        "can_wholesale": {
+          "type": "bool",
+          "default": False,
+          "description": "True/False for if technology has option to export energy that is compensated at the wholesale_rate_us_dollars_per_kwh. Note that a technology can only participate in either net metering or wholesale rates (not both)."
+        },
+        "can_export_beyond_site_load": {
+          "type": "bool",
+          "default": False,
+          "description": "True/False for if technology can export energy beyond the annual site load (and be compensated for that energy at the wholesale_rate_above_site_load_us_dollars_per_kwh)."
+        },
+        "can_curtail": {
+          "type": "bool",
+          "default": False,
+          "description": "True/False for if technology can curtail energy produced."
+        },
+        "macrs_option_years": {
+          "type": "int",
+          "restrict_to": macrs_schedules,
+          "default": 0,
+          "description": "MACRS schedule for financial analysis. Set to zero to disable"
+        },
+        "macrs_bonus_pct": {
+          "type": "float",
+          "min": 0.0,
+          "max": 1.0,
+          "default": 0.0,
+          "description": "Percent of upfront project costs to depreciate under MACRS"
+        }        
+      },
+      "GHP": {
+        "force_ghp": {
```

Comment:
"force" seems like the wrong term: maybe "require_ghp_purchase" or "require_ghp_install" ?